addons/project/static/src/components/project_control_panel/project_control_panel.xml

```diff
@@ -2,15 +2,15 @@
 <templates xml:space="preserve">
 
     <t t-name="project.ProjectControlPanelContentBadge" owl="1">
-        <t t-tag="isProjectUser ? 'button' : 'span'" class="badge border d-flex p-2 ms-2 bg-view" data-hotkey="y">
+        <t t-tag="isProjectUser ? 'button' : 'span'" class="btn border badge d-flex p-2 align-items-center" data-hotkey="y">
```

Comment:
```suggestion
        <button class="btn border badge d-flex p-2 bg-view align-items-center" data-hotkey="y">
```
The check here is actually not needed so you can remove it and set the tag to a button. 
I am also not sure why you removed the ```bg-view``` class. What was the reason ?

Reply:
> The check here is actually not needed so you can remove it and set the tag to a button. I am also not sure why you removed the `bg-view` class. What was the reason ?

Now i add bg-view class

If we always add the button, then the none project user will also see the button and the  :hand: icon will appear on the button hover, but the action will not be performed, so we should not always add the button.

Reply:
Thanks for the reply. 
I said that the check is not needed because in fact, non project users are not able to access the project status from the kanban view - thus they would not reach this check in the first place. If you have a use case / flow where a non project user does reach this check, let me know !

Reply:
> Thanks for the reply. I said that the check is not needed because in fact, non project users are not able to access the project status from the kanban view - thus they would not reach this check in the first place. If you have a use case / flow where a non project user does reach this check, let me know !

I guessed from the code. But now I checked that non project user can't access project update. So you are right I have updated the code as per your suggestion